app/controllers/catalog_controller.rb

```diff
@@ -64,7 +64,7 @@ class CatalogController < ApplicationController
     config.index.title_field = 'normalized_title_ssm'
     config.index.display_type_field = 'level_ssm'
     config.index.document_component = Arclight::SearchResultComponent
-    config.index.group_component = Arclight::GroupComponent
+    config.index.group_component = Ngao::Arclight::GroupComponent
```

Comment:
hm, i think this may impact the ticket i'm working on.https://github.com/scientist-softserv/archives_online/issues/24